### PR TITLE
added cart additions and cleaned up formatting

### PR DESCRIPTION
**Pull request by RyanTFulmer**

### CODE REVIEW

File: src/components/overview/OverviewMain.jsx [hunk 1]
```diff
@@ -134,11 +134,11 @@ class overviewMain extends React.Component {
   render() {
     return (
       <div>
-        <div class="level">
-          <p class="level-left">
+        <div class="columns">
```
Comment (dprevoznik) [inline src/components/overview/OverviewMain.jsx:137]: Interested in your thought process of switching from level to columns. Did you notice limitations of using Bulma's level styling?

File: src/components/overview/sub-Components/AddToBag.jsx [hunk 2]
```diff
@@ -1,7 +1,56 @@
...
+
 const AddToBag = function() {
-  return <div></div>;
+  return (
+    <div>
+      <div>
+        <div class="select is-medium">
+          <select>
+            <option>choose size</option>
+            {sizeTypes.map(eachSize => {
```
Comment (dprevoznik) [inline src/components/overview/sub-Components/AddToBag.jsx:34]: These are really descriptive variable names. Makes understanding what is happening much simpler.

File: src/components/overview/sub-Components/BasicDetails.jsx [hunk 3]
```diff
@@ -20,9 +20,9 @@ const BasicDetails = function(props) {
     return (
       <section class="section">
         <div class="container">
-          <div>{props.state.basicDetails.category}</div>
-          <div>{props.state.basicDetails.name}</div>
-          <div>{prices}</div>
+          <h5>{props.state.basicDetails.category}</h5>
```
Comment (dprevoznik) [inline src/components/overview/sub-Components/BasicDetails.jsx:23]: Posssibly consider using object destructuring in your function parameters to have less lengthy variable usage (i.e. "props.state.basicDetails.category")

File: src/components/overview/OverviewMain.jsx (hunk 1 above)
Comment (RyanTFulmer) [reply src/components/overview/OverviewMain.jsx:137]: still trying to figure things out! it's hard to get used to

**Review by dprevoznik — COMMENTED** [review]
Ryan this looks great. Seems like you are picking up Bulma mechanics at a rapid pace. Let's talk when you get to more advanced stuff and need to customize it.

**Pull request merged**